src/components/chats/hooks/useUnreadCount.ts

```diff
@@ -0,0 +1,35 @@
+import { getCommentIdsByPostIdFromChainQuery } from '@/services/subsocial/commentIds'
+import { getUnreadCountQuery } from '@/services/subsocial/datahub/posts/query'
+import { getDatahubConfig } from '@/utils/env/client'
+import { useMemo } from 'react'
+
+export default function useUnreadCount(chatId: string, lastReadId: string) {
+  const isDatahubEnabled = !!getDatahubConfig()
+
+  const { data: unreadCountFromDatahub } = getUnreadCountQuery.useQuery(
+    { chatId: chatId, lastRead: { postId: lastReadId } },
+    {
+      enabled: isDatahubEnabled && !!lastReadId,
+    }
+  )
+
+  const { data: messageIds } = getCommentIdsByPostIdFromChainQuery.useQuery(
```

Comment:
use if please!

Reply:
The problem is, I can't use if because it's using hooks, and it can't be called inside if
but ill make refactor it

Reply:
refactored